src/test/java/systemtests/AddCommandSystemTest.java

```diff
@@ -187,6 +187,139 @@ public void add() throws Exception {
         assertCommandFailure(command, Tag.MESSAGE_TAG_CONSTRAINTS);
     }
 
+    @Test
+    public void addAlias() throws Exception {
```

Comment:
Instead of lumping everything into one test (if this test fails, it becomes hard to narrow down where the bug is), you can split them into multiple test cases with the format `FeatureUnderTest_scenarioBeingTested_expectedOutcome()`. e.g., `isAnyNull_nullList_returnsTrue()`.